docs/mkdocs.yml

```diff
@@ -77,6 +77,7 @@ nav:
       - webknossos-py/examples/apply_merger_mode.md
       - webknossos-py/examples/learned_segmenter.md
       - webknossos-py/examples/skeleton_synapse_candidates.md
+      - webknossos-py/examples/calculate_segment_sizes.md
```

Comment:
We could group those as `Annotation Examples` or sth similar, but also fine as-is for now.